command.py

```diff
@@ -78,11 +66,13 @@
     'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf8x'}, # lon, lat, alt
     'tare_IMU':                 {'cmd':131, 'type':'fff'},
     'set_IMU':                  {'cmd':132, 'type':'fff'},
-    'set_KDValues':             {'cmd':133, 'type':'fffffff'},
-    'set_KPValues':             {'cmd':134, 'type':'fffffff'},
-    'set_KIValues':             {'cmd':135, 'type':'fffffff'},
+    
     'update_waypoint':          {'cmd':136, 'type':'ddffBxxB'}, # lon, lat, alt, rad, type, id
-    'set_gains':                {'cmd':137, 'type':'hfff'} #first int represents the channel #, 3 floats are KD, KP, KI
```

Comment:
I dont understand what you mean. This is alot more friendlier than having to remember the gain channel mapping, and I don't get whats so bad about doing it this way